Doc/deprecations/pending-removal-in-3.16.rst

```diff
@@ -1,18 +1,42 @@
 Pending Removal in Python 3.16
 ------------------------------
 
+* :mod:`builtins`:
+
+  * Bitwise inversion on Boolean types, ``~True`` or ``~False``
+    has been deprecated since Python 3.12,
+    as it produces surprising and unintuitive results (``-2`` and ``-1``).
+    Use ``not x`` instead for the logical negation of a Boolean.
+    In the rare case that you need the bitwise inversion of
+    the underlying integer, convert to ``int`` explicitly (``~int(x)``).
+
 * :mod:`array`:
-  :class:`array.array` ``'u'`` type (:c:type:`wchar_t`):
-  use the ``'w'`` type instead (``Py_UCS4``).
 
-* :mod:`builtins`:
-  ``~bool``, bitwise inversion on bool.
+  * The ``'u'`` format code (:c:type:`wchar_t`)
+    has been deprecated in documentation since Python 3.3
+    and at runtime since Python 3.13.
+    Use the ``'w'`` format code (:c:type:`Py_UCS4`)
+    for Unicode characters instead.
+
+* :mod:`shutil`:
+
+  * The :class:`!ExecError` exception
+    has been deprecated since Python 3.14.
+    It has not been used by any function in :mod:`!shutil` since Python 3.4,
+    and is now an alias of :exc:`RuntimeError`.
 
 * :mod:`symtable`:
-  Deprecate :meth:`symtable.Class.get_methods` due to the lack of interest.
-  (Contributed by Bénédikt Tran in :gh:`119698`.)
 
-* :mod:`shutil`: Deprecate :class:`!shutil.ExecError`, which hasn't
-  been raised by any :mod:`!shutil` function since Python 3.4. It's
-  now an alias for :exc:`RuntimeError`.
+  * The :meth:`Class.get_methods <symtable.Class.get_methods>` method
+    has been deprecated since Python 3.13.
```

Comment:
```suggestion
    has been deprecated since Python 3.14.
```

Actually, we fixed it in 3.12, 3.13 and 3.14 but immediately deprecated it in 3.14 (the reason being really the lack of interest).